src/library/scala/collection/immutable/Range.scala

```diff
@@ -479,7 +479,7 @@ object Range {
 
   // As there is no appealing default step size for not-really-integral ranges,
   // we offer a partially constructed object.
-  class Partial[T, U](private val f: T => U) extends AnyVal {
+  private[scala] class Partial[T, U](private val f: T => U) extends AnyVal {
```

Comment:
This needs to be public. Numeric type implementations must be able to call and return `Partial`. In my experience having `private[scala]` on an element that is not directly in package `scala` is often a bad idea. If we can't restrict its usage to its own package, user code may want to call it, too.